cloudify_hostpool/rest/service.py

```diff
@@ -0,0 +1,69 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+import httplib
+
+from flask import Flask
+from flask import jsonify
+from flask_restful import Api
+
+import cloudify_hostpool.exceptions
+
+
+app = Flask(__name__)
+api = Api(app)
+
+
+@app.errorhandler(cloudify_hostpool.exceptions.ByonHTTPException)
+def handle_byon_errors(error):
+    response = jsonify(error.to_dict())
+    response.status_code = error.get_code()
+    return response
+
+
+@app.route('/servers', methods=['GET'])
+def list_servers():
+    """ List allocated servers"""
+    server = {'function': 'list_servers'}
+    return jsonify(server), httplib.NOT_FOUND
```

Comment:
Currently there is no 'binding'. I've created an interface that will handle all these requests (is in another branch: https://github.com/cloudify-cosmo/cloudify-host-pool-service/tree/cfy-2108_rest_backend_interface ).